### PR TITLE
Updating cadence ops with new name space, rebasing 6 optimized ops

### PR DESCRIPTION
**Pull request by dijopaul**
- Adding 'cadence' name space in all ops
- Rebasing all ops add,sub,div,mul,sigmoid,tanh as there are major merge conflicts
- The ops are already reviewed under _https://github.com/pytorch/executorch/pull/5483/_

### CODE REVIEW

File: backends/cadence/aot/functions_hifi.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 - op: add.out
   kernels:
     - arg_meta: null
-      kernel_name: torch::executor::add_out
+      kernel_name: impl::HiFi::add_out
```
Comment (mcremon-meta) [inline backends/cadence/aot/functions_hifi.yaml:28]: probably should be cadence::impl::HiFi too, but we can do that in a follow up if this can be merged

**Review by mcremon-meta — APPROVED** [review]

File: backends/cadence/aot/functions_hifi.yaml (hunk 1 above)
Comment (dijopaul) [reply backends/cadence/aot/functions_hifi.yaml:28]: I will create a new PR for this

Comment (dijopaul) [reply backends/cadence/aot/functions_hifi.yaml:28]: https://github.com/pytorch/executorch/pull/6433

**Comment by kirklandsign** [comment]
Hi @dijopaul sorry I reverted this because it seems to break our internal CI. Maybe we need to import to internal repo first and run CI there

**Comment by kirklandsign** [comment]
xplat/executorch/backends/cadence/hifi/operators/op_tanh.cpp:10:10: fatal error: 'executorch/kernels/portable/cpu/pattern/pattern.h' file not found
#include <executorch/kernels/portable/cpu/pattern/pattern.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by kirklandsign** [comment]
@mcremon-meta need to update BUCK

**Comment by zonglinpeng** [comment]
@kirklandsign https://github.com/pytorch/executorch/pull/6477 the fix is landing internally. thanks for the heads up. cc. @mcremon-meta

**Pull request merged**